src/søknad/barna-dine/LeggTilBarn.less

```diff
@@ -0,0 +1,71 @@
+
+.legg-til-barn {
+    .fødselsnummer {
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-start;
+        margin-top: 3rem;
+    }
+
+    .bo-hos-deg-spørsmål {
+        margin-top: 2rem;
+    }
+
+    .legg-til-barn-knapp {
+        display: block;
+        margin: 0 auto;
+        margin-top: 5rem;
+    }
+
+    .personnummer {
+        margin-left: 2rem;
+    }
+
+    .termindato {
+        margin-top: 4rem;
+    }
+
+    h2 {
+        text-align: center;
+        margin-bottom: 2rem;
+    }
+
+    .input-navn {
+        max-width: 400px;
+        margin-top: 4rem;
+    }
```

Comment:
I stedet for masse margin og paddings lagt inn i hver spesifikke css klasse, kunne vi brukt <SeksjonGruppe/>, <KomponentGruppe/> eller <FeltGruppe/> for å få et mer konsis styling av mellomrom og sånt? 
De kan selvfølgelig tweakes litt, men tanken er da at f.eks. `<KomponentGruppe> {innhold} </KomponentGruppe>`, vil gi alle utenom det siste barnet av Komponentgruppe padding-bottom: 50px :) 

Burde egentlig kanskje endret til rem i stedet for px når jeg tenker meg om 😅

Reply:
Jepp, la til KomponentGruppe :)